build_tools/services.rb

```diff
@@ -9,10 +9,10 @@ class ServiceEnumerator
     MANIFEST_PATH = File.expand_path('../../services.json', __FILE__)
 
     # Minimum `aws-sdk-core` version for new gem builds
-    MINIMUM_CORE_VERSION = "3.216.0"
+    MINIMUM_CORE_VERSION = "3.225.0"
 
     # Minimum `aws-sdk-core` version for new S3 gem builds
-    MINIMUM_CORE_VERSION_S3 = "3.216.0"
+    MINIMUM_CORE_VERSION_S3 = "3.225.0"
```

Comment:
Self-Reminder to double check before merging

Reply:
Keep in mind that STS, SSO, etc will also cause this to increase, so you cannot select a patch version, only a minor version.